pkg/scheduler/api/devices/nvidia/vgpu/device_info.go

```diff
@@ -172,6 +203,18 @@ func (gs *GPUDevices) SubResource(pod *v1.Pod) {
 					gs.Device[index].UsedMem -= uint(deviceused.Usedmem)
 					gs.Device[index].UsedNum--
 					gs.Device[index].UsedCore -= uint(deviceused.Usedcores)
+					gs.Device[index].PodMap[pod.Name].UsedCore -= uint(deviceused.Usedcores)
```

Comment:
The metrics name is a little confused, it's not clear the metric represents the allocated resources or total resources，maybe we can use a more straight name.